src/components/LandingSection.js

```diff
@@ -0,0 +1,155 @@
+import React, { useState } from "react";
+import {
+  Grid,
+  Row,
+  Column,
+  TextInput,
+  Button,
+  ToastNotification,
+} from "carbon-components-react";
+
+const LandingSection = () => {
+  const [error, setError] = useState("");
+  const [id, setId] = useState("");
+  const [showToaster, setShowToaster] = useState(false);
+
+  const handleBtnClose = () => {
+    setShowToaster(false);
+  };
+
+  const checkInput = (val) => {
+    setError("");
+    setId(val);
+  };
+
+  function isAlphanumeric(str) {
+    return /^[a-zA-Z0-9]+$/.test(str);
+  }
+
+  const submitID = (e) => {
+    e.preventDefault();
+    if (!isAlphanumeric(id)) {
+      setError("Only alphanumeric input is allowed");
+    } else {
+      setError("");
+      setId("");
+      setShowToaster(true);
+    }
+  };
+
+  return (
+    <Grid data-testid="landing-section-container" className="hero__container">
+      <Row className="hero__content">
+        <Column className="hero__text">
+          <div className="hero__section">
+            <h1>
+              Welcome to <span>Patient Portal</span> App
+            </h1>
+            <p>Patient-Centric Care Made Accessible</p>
+            <p>
+              Download and Track all your <span>Medical Records</span> with ease
+            </p>
+            <div className="hero__form">
+              <p>
+                Enter your hospital ID to <span>get started</span>
+              </p>
+              <div className="input__section">
+                <TextInput
+                  data-testid="text-input-desktop"
+                  id="text-input-1"
+                  labelText=""
+                  value={id}
+                  type="text"
+                  onChange={(event) => checkInput(event.target.value)}
+                />
+                <p className="error__text">{error}</p>
+                <Button
+                  data-testid="otp-btn-desktop"
+                  className="otp__button primary__button"
+                  onClick={submitID}
+                >
+                  Get OTP
+                </Button>
+              </div>
+            </div>
+          </div>
+        </Column>
+        <Column className="hero___imagesection">
+          <img
+            className="hero__image"
+            src="https://i.ibb.co/rkw0fjk/Header-img.png"
```

Comment:
I have used these images from Pexels which does not require attribution for commercial images : https://help.pexels.com/hc/en-us/articles/360042295174-What-is-the-license-of-the-photos-and-videos-on-Pexels-. I had uploaded them to a cloud provider for the time being as Alpit would fork the same repo and won't be needing these images. So should add an assets folder and move these images there and he could delete them, or do this in the next PR?

Reply:
Yes. Better to then download and copy these to assets folder, and mention there in a README.md in assets folder, the source of the images and that these can be packaged with no attribution required. Better to package as part of codebase sub-folder, so that the external links don't suddenly become invalid. If these are required by another sub-project, they can create their own assets folder.

Reply:
Okay, will make these changes.